kuma/wiki/api/v1/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.conf.urls import url
+
+from kuma.wiki.views import document
+
+
+urlpatterns = [
+    url(r'^doc/(?P<locale>[^/]+)/(?P<path>.*)$',
```

Comment:
Perhaps `slug` would be a better name than `path`, only because it matches our current usage?

Reply:
Done.

Reply:
Done